puphpet/config.yaml.template

```diff
@@ -60,10 +71,20 @@ vagrantfile-local:
         host: detect
 server:
     install: '1'
+    apt-ppa-repos:
+        - ppa:git-core/ppa   # Provides newer git-core. -bp
+        - ppa:chris-lea/node.js  # Provides newer node and npm. -bp
     packages:
-        - vim
-        - memcached
         - curl
+        - emacs
```

Comment:
Read my comments. No they aren't, but it's easier to trim down than to hunt for the package names you need. 

Sent from my iPhone

> On Jan 31, 2015, at 8:49 AM, Andrew Lechowicz <notifications@github.com> wrote:
> 
> In puphpet/config.yaml.template:
> 
> >          - curl
> > +        - emacs
> Are all of these really needed in every project?
> 
> —
> Reply to this email directly or view it on GitHub.
>